src/test/java/spoon/generating/MetamodelGenerator.java

```diff
@@ -19,6 +19,9 @@
 
 public class MetamodelGenerator {
 
+	private MetamodelGenerator() {

```

Comment:
I don't see the point here of creating a private constructor instead of the implicit public one?

Reply:
Checkstyle requires utility classes (helper class with only static methods) to have only private ctr to prevent creating objects.

I think it is very practical and useful rule. Every time I write utility, I prevent instantiation.

Reply:
Btw. I have seen many times invoking static methods by object :-O
new Utils().getInt()
Terrible!

Reply:
@surli 
I am ready to merge.

Reply:
Could you merge it, so we can close this refactorization task? :-)